src/util/pure.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import React from 'react';
+
+/**
+ * A simple helper function which provides access to the pure component.
+ *
+ * Components itself might be pure, but are modified by using the `subscribeStores` function. However, this makes
+ * testing, especially `shallow rendering` more difficult, so a helper API
+ * which provides the required access is necessary.
+ *
+ * @param {Object} connectedParent The connected parent which will be generated by the `subscribeStores` function.
+ * @param {Object} props           The props for the function to test.
+ *
+ * @returns {React.Element} A newly created react.js element.
+ */
+export default (connectedParent, props) => React.createElement(connectedParent.wrapped(), props);
```

Comment:
we should check whether the `wrapped()` method exists and throw a better message after that. This isn't always the case, if a component is not decorated.